### PR TITLE
"Fixed" bloom being dark when zooming out

### PR DESCRIPTION
**Pull request by ado1928**
My attempt at making the bloom better

### CODE REVIEW

File: core/src/io/anuke/mindustry/graphics/Bloom.java [hunk 1]
```diff
@@ -331,7 +331,7 @@ private void gaussianBlur(){
      * @param intensity multiplier for blurred texture in combining phase. must be
      * positive.
      */
-    public void setBloomIntesity(float intensity){
+    public static void setBloomIntesity(float intensity){
```
Comment (Anuken) [inline core/src/io/anuke/mindustry/graphics/Bloom.java]: This is a very bad idea. **Never** make things static unless they need to be, especially in a case like this where the bloom can be uninitialized.

File: core/src/io/anuke/mindustry/core/Renderer.java [hunk 2]
```diff
@@ -104,7 +104,9 @@ public void update(){
         Color.white.set(1f, 1f, 1f, 1f);
 
         camerascale = Mathf.lerpDelta(camerascale, targetscale, 0.1f);
-
+        
+        Bloom.setBloomIntesity(12f / camerascale);
```
Comment (Anuken) [inline core/src/io/anuke/mindustry/core/Renderer.java]: What if bloom is disabled, and/or bloom doesn't work on someone's machine?

Comment (ado1928) [reply core/src/io/anuke/mindustry/core/Renderer.java]: I think that this whole issue about bloom could be fixed by adjusting Gaussian blur size based on zoom, but that would need a different code for the Gaussian shader. Also, by disabling the original layer and displaying only the Gaussian layer, I've noticed that its much darker than the original, is this intentional or just an issue?

Comment (Anuken) [reply core/src/io/anuke/mindustry/core/Renderer.java]: That's either a side effect of the blurring taking into account nearby black clear pixels, or just how the threshold filter outputs the result.

**Comment by Anuken** [comment]
Closing until a better solution is found.

**Pull request closed without merging**